diff --git a/node_modules/react-native-hooks/lib/useKeyboard.js b/node_modules/react-native-hooks/lib/useKeyboard.js
index 1264896..b28b407 100644
--- a/node_modules/react-native-hooks/lib/useKeyboard.js
+++ b/node_modules/react-native-hooks/lib/useKeyboard.js
@@ -8,8 +8,8 @@ export default function useKeyboard() {
     setKeyboard({
       isKeyboardShow: true,
       Keyboard,
-      endCoordinates: e.endCoordinates,
-      startCoordinates: e.startCoordinates
+      endCoordinates: e ? e.endCoordinates : null,
+      startCoordinates: e ? e.startCoordinates : null
     })
   }
 
@@ -17,8 +17,8 @@ export default function useKeyboard() {
     setKeyboard({
       isKeyboardShow: false,
       Keyboard,
-      endCoordinates: e.endCoordinates,
-      startCoordinates: e.startCoordinates
+      endCoordinates: e ? e.endCoordinates : null,
+      startCoordinates: e ? e.startCoordinates : null
     })
   }
 
